### PR TITLE
docs: applied a much straight forward sentence

### PR DESCRIPTION
**Pull request by jrumandal**
To understand the current sentence it requires to read it multiple times. This fix should help developers to understand the sentence in a quicker way.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Displays the updated paragraph

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by petebacondarwin — APPROVED** [review]

File: aio/content/guide/singleton-services.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ For a sample app using the app-wide singleton service that this page describes,
 
 There are two ways to make a service a singleton in Angular:
 
-* Declare `root` for the value of the `@Injectable()` `providedIn` property
+* Set `root` as value in the `@Injectable()`'s `providedIn`' property
```
Comment (jbogarthyde) [inline aio/content/guide/singleton-services.md]: I suggest :
Set the `providedIn` property of the `@Injectable()` to `root`.

**Review by jbogarthyde — CHANGES_REQUESTED** [review]
We can make this even more straight-forward :-)

File: aio/content/guide/singleton-services.md (hunk 1 above)
Comment (kara) [reply aio/content/guide/singleton-services.md]: +1 to this

**Review by mhevery — APPROVED** [review]

**Comment by atscott** [comment]
Hi @jrumandal, can you rebase your PR so that it can be merged and also squash the two commits into one with the message being something like "docs: reword explanation for using singleton services to be more understandable" or something similar?

**Comment by jrumandal** [comment]
> Hi @jrumandal, can you rebase your PR so that it can be merged and also squash the two commits into one with the message being something like "docs: reword explanation for using singleton services to be more understandable" or something similar?

Hi, is that okay? :)

**Comment by atscott** [comment]
@jrumandal - commit looks good now, thank you! In order to merge, you also need to rebase with master. You can find the directions for this in the note at the end of the CI failure in the setup step:

>Rebase instructions for PR Author, please run the following commands:
> git fetch upstream master;
> git checkout patch-7;
> git rebase upstream/master;
> git push --force-with-lease;

**Comment by jrumandal** [comment]
@atscott  Done ^^

**Pull request closed without merging**